ecommerce/views.py

```diff
@@ -114,6 +115,8 @@ def post(self, request, *args, **kwargs):
                     )
         else:
             order.status = Order.FULFILLED
+            post_payment(order)
```

Comment:
What should happen if `post_payment` fails? It seems like the way it is the order would remain with status `CREATED`. If the `post_payment` step is not essential we should keep the order as `FULFILLED` and log the exception so we get a Sentry report. If it's essential we should mark the order as `FAILED`. However I don't think we should do that since we already have their money at that point.

Reply:
Changed to catch any exception that occurs during the post_payment function and log it.